src/appengine/handlers/cron/oss_fuzz_build_status.py

```diff
@@ -185,19 +184,26 @@ def send_reminder(issue_tracker, issue_id, build_id):
 class Handler(base_handler.Handler):
   """Build status checker."""
 
-  def _close_fixed_builds(self, build_status, build_type):
+  def _close_fixed_builds(self, projects, build_type):
     """Close bugs for fixed builds."""
     issue_tracker = issue_tracker_utils.get_issue_tracker()
     if not issue_tracker:
       raise OssFuzzBuildStatusException('Failed to get issue tracker.')
 
-    for build in build_status['successes']:
-      project_name = build['name']
-      build_failure = get_build_failure(project_name, build_type)
+    for project in projects:
+      project_name = project['name']
+      builds = project['history']
+      if not builds:
+        continue
 
+      build_failure = get_build_failure(project_name, build_type)
       if not build_failure:
         continue
 
+      build = builds[0]
```

Comment:
while trying to understand why this part is needed, I realized that we might want to run the cron more often now, given that more frequent builds are supported. It runs daily, we can probably run it 4 times a day now?

Reply:
Sure, feel free to make it 4 times.